shakenfist/client/main.py

```diff
@@ -634,6 +634,10 @@ def instance_create(ctx, name=None, cpus=None, memory=None, network=None, networ
     if len(disk) < 1 and len(diskspec) < 1:
         print('You must specify at least one disk')
 
+    if memory < 256:
+        print('WARNING: Assuming you have specified memory in GB\n')
+        memory *= 1024
+
```

Comment:
This seems reasonable to me, but maybe we should file an issue to remove the special casing in a later release? v0.3 perhaps?

Reply:
In fact, have https://github.com/shakenfist/shakenfist/issues/140

Reply:
I only put it in to avoid you wasting another sunday wondering why VM's don't start. :-)